packages/context-entries/lib/Keys.ts

```diff
@@ -298,3 +307,18 @@ export const KeysRdfJoin = {
    */
   lastPhysicalJoin: new ActionContextKey<string>('@comunica/bus-rdf-join:lastPhysicalJoin'),
 };
+
+export const KeysTrackableStatistics = {
+  /**
+   * All discovered links during traversal (not only the relevant ones) in form of parent, child
```

Comment:
```suggestion
   * All discovered links during query execution. Not all of them will necessarily be dereferenced.
```

(the reader should be able to find additional information in `StatisticLinkDiscovery`)